slotfills/sections/radio.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { PanelRow, RadioControl } from '@wordpress/components';
+import Field from '@/interfaces/field';
+
+interface RadioProps {
+  field: Field;
+  index?: number;
+  label?: string;
+  valueHook: (key: number | string) => [any | any[], Function];
```

Comment:
```suggestion
  valueHook: (key: number | string) => [any, Function];
```
I believe that `any` covers `any | any[]` - Would it be possible not to use `any` here?